### ISSUE TITLE
Feature Request: cache.all method

### ISSUE THREAD

**Issue by OzairP** [royriojas/flat-cache#15]
I think it would be nice to fetch the whole cache object.

```JS
const fcache = require('flat-cache')

const birthdays = fcache.load('birthdays')

birthdays.set('Alice', '01/01/98')
birthdays.set('Bob', '02/02/97')

birthdays.all() // returns { 'Alice': '01/01/98', 'Bob': '02/02/97' }
```

**Comment by royriojas** [comment]
Hi @OzairP,

I don't think of a reason to no add this one, I will try to find some time to add a method to return the cache object, but if you're interested into adding it, PRs are welcome!

Kind regards

**Comment by OzairP** [comment]
Awesome, can you push the newest changes to NPM whenever possible? Thanks!

**Comment by royriojas** [comment]
Thanks @OzairP. Published v1.3.0
